src/GlobalState.lua

```diff
@@ -90,6 +95,36 @@ function M.shuffle(list)
   end
 end
 
+function M.logistic_scan()
+  local entities = game.surfaces[1].find_entities_filtered{name = {"logistic-chest-requester", "logistic-chest-buffer"}}
+  for _, ent in ipairs(entities) do
+    M.logistic_add_entity(ent)
+  end
+end
+
+function M.logistic_get(unit_number)
+  return global.mod.logistic[unit_number]
+end
+
+function M.logistic_add_entity(entity)
+  if global.mod.logistic[entity.unit_number] == nil then
+    global.mod.logistic[entity.unit_number] = entity
+    Queue.push(global.mod.scan_queue, entity.unit_number)
+  end
+end
+
+function M.logistic_del_entity(entity)
```

Comment:
Is there any difference between this an `M.logicstic_del` besides using entity vs unit_number? Can this be changed to only use `M.logistic_del`?

Reply:
Sure, I assume you'd be OK doing that? Easy enough to change the caller to use `entity.unit_number`